ODIN_II/SRC/memories.cpp

```diff
@@ -35,6 +35,9 @@
 #include "partial_map.h"
 #include "vtr_util.h"
 #include "vtr_memory.h"
+#include <iostream>
```

Comment:
I see you only used file stream, so in this case please avoid using extra includes like `iostream`